src/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.cs

```diff
@@ -522,13 +513,15 @@ public override Module Module
             }
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
This TODO is not applicable. It would be better to change this to return `EmptyCAHolder` like the other places

Reply:
Fixed.